exercises/matrix/.meta/src/reference/java/Matrix.java

```diff
@@ -1,11 +1,11 @@
 import java.util.regex.Pattern;
 
-public class Matrix {
+ class Matrix {
```

Comment:
I think all the lines you've changed you've indented one space too much? Try to keep the indentation the same as it was :)

Reply:
Sure !